app/routes/events/components/StripeElement.tsx

```diff
@@ -57,7 +57,9 @@ type CompleteStatus =
 const StripeElementStyle = {
   style: {
     base: {
-      color: '#0d0d0d',
+      color: window.matchMedia('prefers-color-scheme: dark')
+        ? '#f2f2f2'
+        : '#0d0d0d',
```

Comment:
`color` should just be set equal to `var(--lego-font-color)` (which will also fix the bug).

Reply:
This will unfortunately not work since the input is actually inside an `<iframe>`. It uses a `CardNumberElement` from `react-stripe`, which handles the styling.
![image](https://github.com/webkom/lego-webapp/assets/33326578/e67a5a29-95e3-4218-ba3e-5bf58cf9f5a1)

Reply:
You can still override the styling. You may just add it to the classname though, that's better

Reply:
The `className` prop only applies the class to the element container (outside the `<iframe>`).
Refering to this comment: https://github.com/stripe-archive/react-stripe-elements/issues/51#issuecomment-320147662

Reply:
@ivarnakken